classes/module-signing.bbclass

```diff
@@ -44,3 +62,6 @@ addtask sign_modules after do_install before do_package
 do_install[lockfiles] = "${TMPDIR}/kernel-scripts.lock"
 # Explicit keys sign modules in do_sign_modules
 do_sign_modules[lockfiles] = "${TMPDIR}/kernel-scripts.lock"
+
+do_sign_modules[depends] += "virtual/kernel:do_shared_workdir"
```

Comment:
do_sign_modules already transitively depends on "virtual/kernel:do_shared_workdir".  Are you adding the explicit depends for SIGN_FILE?

Reply:
Yes, and you are correct. Also `get_signing_cert` in `file-checksums` is already satisfied from `do_configure` dependencies.